tools/rcverify.sh

```diff
@@ -277,7 +277,7 @@ CMP=$(eval "$CMD")
 validate $? 0 "$CMD"
 
 printf "verifying sources have proper headers..."
-if [ -f "$DIR/$BASE/tools/travis/scancodeExlusions" ]; then
+if [ -f "$DIR/$BASE/tools/travis/scancodeExclusions" ]; then
```

Comment:
This is OK. I don’t think we use this anywhere (I looked in several repos for a match).

Reply:
Found one reference https://github.com/apache/openwhisk-apigateway/blob/master/tools/travis/scancodeExclusions.